README.md

```diff
@@ -103,9 +103,15 @@ It will output details for each request/response similar to the following snippe
 
 ### HTTP Endpoints
 
-If the model server is up and running, you can access the model server API via `http://localhost:8081/api/v1/*`.
+The Model Server supports two versions of the API (v1 and v2). If the model server is up and running, you can access the model server v1 API via `http://localhost:8081/api/v1/*`,
+and the v2 API via `http://localhost:8081/api/v2/*`
```

Comment:
The readme is showing the current state. And we have 2 versions at the moment.
As long as we have no V2 Java Client we need both APIs.
At a near future we will have only one API and thus deprecate V1 and remove it at some point.

Reply:
Alright, let's hope that "in the near future" is not too far away. And since there is a proper documentation task #170 for v2 I think it is fine to leave the README like that for now. Still, I believe it might be a bit confusing for others why there are two versions of the API all of a sudden.